src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier randomColour = new ColorSupplier();
+        Random random = new Random();

```

Comment:
you need to declare fields in this class to hold instances of `Random` and `ColorSupplier`:
```
private final Random random = new Random();
private final ColorSupplier colorSupplier = new ColorSupplier();
```

this way you won't have to create new `Random` and `ColorSupplier` every time this method is called